### DISCUSSION TITLE
API bug [May 08]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2054] [Discord Threads]
> 🙋‍♂️ **Clement**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### please create an issue in github and we'll take a look  
#  
> 🙋‍♂️ **Clement**  
#### Gonna link it here after  
#  
> 👨‍🚀 **wingkwong**  
#### thanks  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/972438786426429541)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
